cmd/jaeger/internal/exporters/storageexporter/sanitizer/empty_service_name_sanitizer.go

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package sanitizer
+
+import (
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+// Constants for the replacement names
+const (
+	serviceNameReplacement = "empty-service-name"
+	nullProcessServiceName = "null-process-and-service-name"
```

Comment:
```suggestion
	missingServiceName = "missing-service-name"
```
there is no "process" anymore.